test/distributions/test_distributions.py

```diff
@@ -2411,6 +2411,18 @@ def ref_log_prob(idx, x, log_prob):
         self._check_log_prob(Exponential(rate), ref_log_prob)
         self._check_forward_ad(lambda x: x.exponential_())
 
+        def mean_var(lambd, sample):
+            sample.exponential_(lambd)
+            mean = sample.mean()
+            var = sample.var()
+            self.assertEqual((1. / lambd), mean, atol=1e-2, rtol=1e-2)
+            self.assertEqual((1. / lambd) ** 2, var, atol=2e-2, rtol=1e-2)
+
+        for dtype in [torch.float, torch.double]:
```

Comment:
Since your change also handles types that are not double or float, please cover other types too, e.g., bfloat16 and half.